pipeline/00-ingest.R

```diff
@@ -169,7 +169,7 @@ process_array_columns <- function(data, selector) {
   data %>%
     mutate(
       across(
-        selector,
+        !!enquo(selector),
```

Comment:
`tidyselect` deprecated passing a character vector. So now we need to make sure the selector (in this case `starts_with()`) is evaluated within `across()`.